adr/ADR-31.md

```diff
@@ -126,9 +154,11 @@ Direct Get replies contain the message along with the following message headers:
 - `Nats-Subject`: message subject
 - `Nats-Num-Pending`: when batched, the number of messages left in the stream matching the batch parameters
 - `Nats-Last-Sequence`: when batched, the stream sequence of the previous message
-
+- `Nats-UpTo-Sequence`: when doing multi subject gets the sequence should be used for following requests to ensure consistent reads
+- 
```

Comment:
Any need to add the `Status` and `Description` headers or link to where those are defined in other ADRs?

Reply:
As above, not sure where they are defined but they're not real headers as such more ways to get at the response status from the protocol message here:

```
HMSG _INBOX.4NogKOPzKEWTqhf4hFIUJV.yo2tE1ep 1 28 28
NATS/1.0 408 Bad Request
```

Status 408 Description Bad Request...its standard things so not covering here